Script to download tinycore linux.

pnasrat opened the pull request:
Work towards automated testing #24.

Download tinycorelinux from a mirror and patch.

Update hyperkitrun.sh to call script and also explicitly call make if
needed.

Tested commands by using hyperkitrun.sh

Signed-off-by: Pris Nasrat <pnasrat@gmail.com>

ijc25 commented:
LGTM. Thanks!

ijc25 commented on hyperkitrun.sh:
```diff
@@ -20,7 +22,20 @@ LPC_DEV="-l com1,stdio"
+ ./tinycore.sh
+ popd
+fi
 build/com.docker.hyperkit $ACPI $MEM $SMP $PCI_DEV $LPC_DEV $NET $IMG_CD $IMG_HDD $UUID -f kexec,$KERNEL,$INITRD,"$CMDLINE"
```
Would be nice to use the new `$HYPERKIT` here, but I'm not going to let that stop me pushing the merge button.

ijc25 commented on test/tinycore.sh:
```diff
@@ -0,0 +1,21 @@
+# Upstream source is available http://www.tinycorelinux.net/6.x/x86/release/src/
+#BASE_URL="http://www.tinycorelinux.net/"
+
+BASE_URL="http://distro.ibiblio.org/tinycorelinux/"
```
Is this temporary or do we expect `ibiblio.org` to be more reliably available than `tinycorelinux.net`?

ijc25 commented:
I had a couple of late-breaking very minor comments above, but nothing to prevent a merge, thanks!

pnasrat commented:
Thanks! I'll do a follow up when I add a simple automated test which
addresses the comments.

On Wed, Jun 8, 2016 at 5:23 AM, Ian Campbell <notifications@github.com>
wrote:

> I had a couple of late-breaking very minor comments above, but nothing to
> prevent a merge, thanks!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/docker/hyperkit/pull/34#issuecomment-224536006>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AAA15kLMpr_E3ZUsbisVToeZtDfJIEzlks5qJooogaJpZM4IvKUv>
> .
>
